src/accelerate/utils/launch.py

```diff
@@ -172,6 +172,9 @@ def prepare_multi_gpu_env(args: argparse.Namespace) -> Dict[str, str]:
             current_env["FSDP_BACKWARD_PREFETCH"] = str(args.fsdp_backward_prefetch_policy)
         if args.fsdp_state_dict_type is not None:
             current_env["FSDP_STATE_DICT_TYPE"] = str(args.fsdp_state_dict_type)
+        current_env["FSDP_FORWARD_PREFETCH"] = str(args.fsdp_forward_prefetch).lower()
+        current_env["FSDP_USE_ORIG_PARAMS"] = str(args.fsdp_use_orig_params).lower()
+        current_env["FSDP_SYNC_MODULE_STATES"] = str(args.fsdp_sync_module_states).lower()
```

Comment:
Question for you here, are these read/used only by us? Or by FSDP/something specific to it

Reply:
Used only bu us for the FSDP Plugin

Reply:
Got it, eventually we should look at adding the `ACCELERATE_` prefix to keep in line with how the rest of our environmental variables we use/set are, for people to easily see. I can do this in a follow-up PR, no need to worry about this now :)